token/services/config/service_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/config"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestConfigurations tests the Configurations function
+func TestConfigurations(t *testing.T) {
+	// create a new config service by loading the config file
+	cp, err := config.NewProvider("./testdata/token0")
+	assert.NoError(t, err)
```

Comment:
Is there a particular reason why using `assert` and `required` in this test? I personally would always prefer `required` as it calls `t.FailNow()` and thus stops the tests immediately.